synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -971,6 +971,7 @@ CanvasInterface::import_sequence(
 		Layer::Handle layer;
 		modules::lyr_std::Import::Handle curr_layer;
 		rendering::Surface::Handle  cur_surface,prev_surface= rendering::Surface::Handle();
+		rendering::SurfaceResource::LockReadBase cur_lock(curr_layer->rendering_surface);
```

Comment:
You are using `curr_layer` without any assignment/initialization.

Reply:
But I did try initiating the `cur_lock` after the dynamic cast to `curr_layer`. I am still getting the same error.